tests/kernel/boo/conftest.py

```diff
@@ -0,0 +1,16 @@
+from pathlib import Path
+import pytest
+from iree.turbine.kernel.boo.runtime import set_cache_dir, use_cache_dir
+
+
+@pytest.fixture(autouse=True)
+def _isolate_tests(tmp_path):
```

Comment:
nit (unless the pytest `tmp_path` doesn't allow it):
```suggestion
def _isolate_tests(tmp_path: Path):
```

Side note, pretty wild that pytest knows that an arg named `tmp_path` means make a test-specific temporary path (directory?).

Reply:
yea these fixtures are magic, they're useful but also scary😂